runner/worker.go

```diff
@@ -83,6 +85,12 @@ func (w *Worker) makeRequest(tv TickValue) error {
 
 	ctd := newCallData(w.mtd, w.workerID, reqNum, !w.config.disableTemplateFuncs, !w.config.disableTemplateData, w.config.funcs)
 
+	if w.mtd.IsClientStreaming() || w.mtd.IsServerStreaming() {
+		if w.streamInterceptorProviderFunc != nil {
+			w.streamInterceptor = w.streamInterceptorProviderFunc(ctd)
```

Comment:
Is one worker used to make multiple requests? If so, this will keep overriding the previous.